[2021-08-10 02:26:10,316] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-10T02:25:59.333769+00:00 [queued]>
[2021-08-10 02:26:10,330] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_4 2021-08-10T02:25:59.333769+00:00 [queued]>
[2021-08-10 02:26:10,330] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:26:10,331] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-08-10 02:26:10,331] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-08-10 02:26:10,338] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_4> on 2021-08-10T02:25:59.333769+00:00
[2021-08-10 02:26:10,341] {standard_task_runner.py:52} INFO - Started process 219780 to run task
[2021-08-10 02:26:10,348] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_4', '2021-08-10T02:25:59.333769+00:00', '--job-id', '27', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpoka4qsrg', '--error-file', '/tmp/tmp6185vod4']
[2021-08-10 02:26:10,349] {standard_task_runner.py:77} INFO - Job 27: Subtask task_4
[2021-08-10 02:26:10,473] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_4 2021-08-10T02:25:59.333769+00:00 [running]> on host airflowvm
[2021-08-10 02:26:10,592] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_4
AIRFLOW_CTX_EXECUTION_DATE=2021-08-10T02:25:59.333769+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-10T02:25:59.333769+00:00
[2021-08-10 02:26:10,593] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-08-10 02:26:10,594] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-08-10 02:26:10,601] {subprocess.py:75} INFO - Output:
[2021-08-10 02:26:13,603] {subprocess.py:83} INFO - Command exited with return code 0
[2021-08-10 02:26:13,631] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_4, execution_date=20210810T022559, start_date=20210810T022610, end_date=20210810T022613
[2021-08-10 02:26:13,655] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-08-10 02:26:13,662] {local_task_job.py:151} INFO - Task exited with return code 0
